### DISCUSSION TITLE
ComfyUI

### DISCUSSION THREAD

**Discussion by gobshyte** [adieyal/sd-dynamic-prompts#553] [Ideas]
Hi
Does anyone know a way to get this functionality in ComfyUI?

**Comment by gobshyte** [comment]
Ok I think I found it!

https://github.com/WASasquatch/was-node-suite-comfyui

"CLIPTextEncode (NSP): Parse noodle soups from the NSP pantry, or parse wildcards from a directory containing A1111 style wildacrds. "

**Reply by adieyal** [reply]
Last time I looked at ComfyUIs implementation, it was very barebones. If you feel like writing a few lines of  code, you can directly integrate the dynamicprompts library: https://github.com/adieyal/dynamicprompts. It's worth noting that due to how ComfyUI is implemented, some features will be hard to use such as combinatorial mode. Other than that, I managed to get up and running in less than an hour.
